src/style.css

```diff
@@ -0,0 +1,92 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 600;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+form.search-form {
+  margin: 0 auto;
+  position: relative;
+}
+
+form.big-search {
+  width: 100%;
+  height: 70px;
+  margin-bottom: 50px;
+}
+
+form.small-search {
+  width: 100%;
+  height: 42px;
+}
+
+form.big-search::before {
+  content: url("./images/Search.svg");
+  width: 19px;
+  height: 19px;
+  position: absolute;
+  z-index: 1;
+  top: 25px;
+  left: 26px;
+}
+
+form.small-search::before {
+  content: "";
+  background-image: url(./images/Search.svg);
+  background-size: 11px 11px;
+  width: 11px;
+  height: 11px;
+  position: absolute;
+  z-index: 1;
+  top: 15px;
+  left: 13px;
+}
+
+form .search-bar {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  width: 100%;
+  height: 100%;
```

Comment:
it's not quite good.
If you add more components to form, your input will be not what you expect.
input height should not depend on the height of the form.